linz-data-importer/linz_data_importer.py

```diff
@@ -317,6 +322,15 @@ def initGui(self):  # pylint:disable=invalid-name
             help_html.format(self.plugin_dir)
         self.dlg.hHelpHtml.setHtml(help_html.format(icon_path))
 
+        # set about html
+        self.dlg.hAboutHtml.setOpenExternalLinks(True)
+        about_file = os.path.join(self.plugin_dir, "about.html")
+        icon_path = os.path.join(self.plugin_dir, "icons")
```

Comment:
This is the same path as `os.path.join(os.path.dirname(__file__), "icons")` above, so it could be pulled up and reused.